contrib/mesos/pkg/minion/server.go

```diff
@@ -206,17 +206,23 @@ func (ms *MinionServer) launchHyperkubeServer(server string, args []string, logF
 		}
 	}
 
-	// use given environment, but add /usr/sbin to the path for the iptables binary used in kube-proxy
+	// use given environment, but add /usr/sbin and $SANDBOX/bin to the path for the iptables binary used in kube-proxy
 	var kmEnv []string
-	if ms.pathOverride != "" {
-		env := os.Environ()
-		kmEnv = make([]string, 0, len(env))
-		for _, e := range env {
-			if !strings.HasPrefix(e, "PATH=") {
-				kmEnv = append(kmEnv, e)
+	env := os.Environ()
+	kmEnv = make([]string, 0, len(env))
+	for _, e := range env {
+		if !strings.HasPrefix(e, "PATH=") {
+			kmEnv = append(kmEnv, e)
+		} else {
+			if ms.pathOverride != "" {
```

Comment:
the logic in this else block will execute every time we encounter an env var that's not PATH. this is probably not what we want